modelling/src/main/java/org/axonframework/modelling/command/Repository.java

```diff
@@ -60,6 +61,24 @@
      */
     Aggregate<T> newInstance(Callable<T> factoryMethod) throws Exception;
 
+    /**
+     * Creates a new managed instance for the aggregate, using the given {@code factoryMethod} to instantiate the
+     * aggregate's root, and then applying the {@code initMethod} consumer to it to perform (optional) additional
+     * initialization.
+     *
+     * @param factoryMethod The method to create the aggregate's root instance
+     * @param initMethod    The (optional) consumer to initialize the aggregate instance further
```

Comment:
If we make the `initMethod` non-null, we can remove the mention of 'optional'.

Reply:
Removed